README.md

```diff
@@ -1,67 +1,66 @@
+Unpackaged desktop apps</br>(WPF, Win32, WinForms, console) | ❌ Not supported  |                    🔄 [Text rendering](https://docs.microsoft.com/en-us/windows/apps/project-reunion/dwritecore)</br>🔄 [Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview)</br>🔄 [App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111)  |  ✅ [WinUI 3](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md#winui-3)</br>✅ [Text rendering](https://docs.microsoft.com/en-us/windows/apps/project-reunion/dwritecore)</br>✅ [Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview)</br>✅ [App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111) | 
 
-**Compatible**
+✅ Supported | 🔄 Preview
 
-Project Reunion works in all your apps - Win32, Packaged, and UWP - and across many versions of Windows.
 
-**Modern**
+See our [full roadmap](docs/roadmap.md) for a more detailed roadmap.
 
-Project Reunion supports your app's use of modern software libraries for UI, AI, ML, packaging, frameworks, 
-and libraries. Language projections for C++, Rust, C#, and JavaScript expand the benefits to all your 
-apps.  Cloud-backed apps, streaming, and edge-compute apps are all great places to use Project Reunion's 
-capabilities.
 
-**Agile**
 
-Project Reunion ships out of band with OS releases, with regular previews. You get to incrementally 
-adopt Project Reunion components for your existing apps and middleware libraries using NuGet.
+### Version Support
+
+Project Reunion aims to work on supported versions of Windows. Right now we support down to build 17763 (version 1809/October 2018 Update) of Windows 10. We'll use techniques like '_polyfills_' 
+and custom implementations to do the heavy lifting for you and make sure your app works across Windows
+versions and devices.
+
+
+### Documentation
+
+* [More details about Project Reunion](https://github.com/microsoft/ProjectReunion/blob/master/docs/README.md) - 
+More on what the project is and how we're approaching the work.
+* [Roadmap](docs/roadmap.md) - Where we're going. We're following this roadmap that 
+gets us to building a first version of Project Reunion exploring some basic common features.

```

Comment:
Not sure what you're trying to say here. The roadmap covers at least **three** releases this year. Are those all intended to only explore "basic" features?